### ISSUE TITLE
Update zsh-async to 1.8.5 broke slimline on all my accounts

### ISSUE THREAD

**Issue by peti** [mgee/slimline#25]
I'm using slimline via `zplug` on a variety of Linux machines, and with commit b63fa88d80597b6017cc5645b93a069363bb9c7a everything works fine. After updating to de5183f46335fb3a516b5fdf4a24272b18a5963f, however, slimline no longer works. The function `slimline_setup` starts up, but never terminates. I have tried to figure out what the code is doing, but I wasn't able to. I'm not sure how the slimline plugin interacts with zplug exactly. Anyhow, reverting the `async` update fixed slimline on all my accounts.

**Comment by mengelbrecht** [comment]
The `slimline_setup` function (among other things) initializes zsh-async and starts an async worker.
What version of zsh are you using? Does the issue also occur if you try it with zsh 5.8.0?

**Comment by peti** [comment]
Yes, I am using `zsh` version 5.8. That's the version I'm having problems with.

**Comment by mengelbrecht** [comment]
I am unable to reproduce it on a linux machine. Can you share a minimal zsh configuration which reproduces the issue?
